ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -303,15 +295,20 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-gcp-ipi-cco-manual-workload-identity
-- as: gcp-ipi-ovn-ipsec-arm-f14
+- as: gcp-ipi-ovn-ipsec-arm-mixarch-f14
```

Comment:
Shall we use `f28` because it is `f28` in e2e job and we always use `f28` for upgrade job.

Reply:
I am keep the same frequency as before, because we are not adding more missing profile yet.